src/wags_tails/drugbank.py

```diff
@@ -25,12 +26,21 @@ def _get_latest_version() -> tuple[str, str]:
         :return: latest release value and base download URL
         :raise RemoteDataError: if unable to parse version number from releases API
         """
-        releases_url = "https://go.drugbank.com/releases.json"
+        releases_url = "https://go.drugbank.com/releases/latest.json"
         r = requests.get(releases_url, timeout=HTTPS_REQUEST_TIMEOUT)
         r.raise_for_status()
         try:
             latest = r.json()[0]
-            return latest["version"], latest["url"]
+            url = latest["url"]
+            version = (
+                re.match(
+                    r"https:\/\/go.drugbank.com\/releases\/(.*)\/downloads\/all-drugbank-vocabulary",
```

Comment:
is this a regex for just something that looks like a versioning number

Reply:
@mcannon068nw yeah exactly